src/pydvl/parallel/config.py

```diff
@@ -1,21 +1,22 @@
-import logging
+import warnings
 from dataclasses import dataclass
 from typing import Literal, Optional, Tuple, Union
 
 __all__ = ["ParallelConfig"]
 
 
+# TODO: delete this class once it's made redundant in v0.10.0
```

Comment:
won't this get lost? Elsewhere I used
```python
# This string for the benefit of deprecation searches:
# remove_in="0.9.0"
```
since that is the format of the deprecated lib

Reply:
I assumed that it however is doing this would search for a `TODO` I will add your suggestion as well.